### DISCUSSION TITLE
How to create a note with checkboxes

### DISCUSSION THREAD

**Discussion by AndyCoach** [TriliumNext/Trilium#6337] [Q&A] [answered]
Hi, sorry, a non-programmer here. Can't find instructions. Checked CK Editor help, still not a clue HOW to create checkboxes in a Note. Please help..

**Comment by zadam** [comment, marked as answer]
Hello, in a text note, you need to click on the paragraph button and then there's "todo list".

![image](https://user-images.githubusercontent.com/617641/73615859-54886c00-460c-11ea-9dc3-7f550ed7a01e.png)

**Comment by AndyCoach** [comment]
Thank you! I should have guessed, since same approach to moving a table..

**Comment by gerroon** [comment]
@zadam

Is there a way to parse all todos from other pages in page rendered page to somehow get an overlook of what needs to be done?

**Comment by zadam** [comment]
checkboxes are HTML. Parsing with script is possible though not trivial.

**Comment by gerroon** [comment]
I see, well I know nothing about js or html. So that rules out me as the creator of it. 

I was just thinking that a single page can show all todos (unchecked) waiting attention.

**Comment by popey456963** [comment]
@gerroon Might I suggest a new issue for that?
